### PR TITLE
Add ImportDeclaration type

### LINKED ISSUE #70: ImportDeclaration type definition missing
`ImportDeclaration` is defined as a type, but it's missing a declaration.

### PR DESCRIPTION
**Pull request by andremedeiros**
Also, add some linting for Typescript files as this would have been caught by the linter.

This fixes #70

### CODE REVIEW

**Comment by federicobond** [comment]
Now that I see it, `ImportDeclaration` does not describe an existing AST node type, it's an artifact of the ANTLR grammar. It should be removed completely from the types, as it's not used anywhere.

**Comment by andremedeiros** [comment]
I figured but wasn't 100% sure so I looked at the grammar and inferred what was going on there.

I'll update the PR by removing the type. Should we go and merge the rest anyway?

**Comment by federicobond** [comment]
Sure, the rest looks fine.

**Comment by andremedeiros** [comment]
There, force pushed, and updated the commit message. Should be ready to go.

File: index.d.ts [hunk 1]
```diff
@@ -103,7 +102,7 @@ export interface ImportDirective extends BaseASTNode {
   type: 'ImportDirective';
   path: string;
   unitAlias: string,
-  symbolAliases: [string, string][];
+  symbolAliases: Array<[string, string]>;
```
Comment (andremedeiros) [inline index.d.ts]: This was another thing caught by the linter. Can you double check this is correct, please?

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended",
+        "tslint-config-prettier"
+    ],
+    "jsRules": {},
+    "rules": {
+      "interface-name": [true, "never-prefix"]
```
Comment (andremedeiros) [inline tslint.json]: This stops the linter from complaining about interfaces not prefixed with `I`.

**Pull request merged**